static/src/stylesheets/module/_release-message.scss

```diff
@@ -432,7 +432,8 @@ $btn-height: gs-height(1) - $gs-baseline/2; //30px
 .site-message--membership-message,
 .site-message--membership-message-uk,
 .site-message--membership-message-uk-redisplayed,
-.site-message--membership-message-usa {
+.site-message--membership-message-us,
+.site-message--membership-message-us-redisplayed {
```

Comment:
Do we need to have these class names for each edition? I believe the templates have identical styling so they should be good with just generic class names `.site-message--membership-message` and `.site-message--membership-message-redisplayed`.